test/Python/src/FederatedImportLibrary.lf

```diff
@@ -0,0 +1,20 @@
+# Test the new import statement, with the import path enclosed in angle brackets, in federated LF programs
+target Python {
+  timeout: 2 sec
+}
+
+import Count from <lib/Count.lf>
```

Comment:
@tanneberger, tagging you because it still appears to be unclear what should be inside the angular brackets. I thought the notation was reserved for packages, not local paths?

Reply:
This is the old version. I still need to implement what we discussed last time. My focus here was to fix the formatting errors causing the check to fail. I'm currently working on a new version where we shorten the paths inside the angular brackets to `<root/file.lf>` for downloaded Lingo packages and `lib/file.lf` for locally defined libraries

Reply:
Do you mean `"lib/file.lf"` for locally defined libraries? My assumption had been that angular brackets are only for published packages, not locally defined ones.

Reply:
Exactly, use angular brackets for published packages and quotation marks for locally defined libraries. In my previous comment, I was referring to `"lib/file.lf"`, but I mistakenly omitted the quotation marks